app/models/config.rb

```diff
@@ -70,4 +75,17 @@ def self.start_day
     start ||= "monday"
     start.downcase.to_sym
   end
+
+  def validate_urls
+    maybe_url = options.last
+    return if maybe_url.blank?
+    
+    url = UriService.new(maybe_url).uri
+
+    if not url
```

Comment:
Suggestion: use `!` preferrentially to `not` - you can get some nasty bugs from `not`, as its _much_ lower in the order of operations evaluation. I'm not sure I've ever used `not` in over a decade of programming! [See a full explainer here](https://stackoverflow.com/questions/27429711/difference-between-not-and-in-ruby)

Reply:
totally! i went down this rabbit hole with `and` and `&&` and `or` and `||` (lol) but forgot about `not`!